<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homeDialog</class>
 <widget class="QDialog" name="homeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>588</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>143</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16677770</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>160</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="sideMenuVL">
    <item>
     <widget class="QWidget" name="sidebarBackgroud" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(38, 38, 38);border-radius: 15px;</string>
      </property>
      <widget class="QLabel" name="sidebarHome">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>158</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(70, 70, 70);border-radius: 15px;font:15pt;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="sidebarTransactions">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>158</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(70, 70, 70);border-radius: 15px;font:15pt;color:white;</string>
       </property>
       <property name="text">
        <string>Transactions</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="sidebarCalendar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>158</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(70, 70, 70);border-radius: 15px;font:15pt;color:white;</string>
       </property>
       <property name="text">
        <string>Calendar</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="homeTiltle">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>158</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(70, 70, 70);border-radius: 15px;font:15pt;color:white;</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="addAccBtn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>480</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Add Account</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 0, 0);border-radius: 15px;font-size:30px;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QScrollArea" name="accScrollArea">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>701</width>
     <height>171</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>699</width>
      <height>169</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>701</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="accHL"/>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="testPushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>480</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>test</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
